routes/post/load_comments.js

```diff
@@ -1,23 +1,32 @@
 import { Router } from "express";
-import { jwtgetUser } from "../coms/jwtgetUser";
+import jwtgetUser from "../jwtauth/jwtgetUser";
+import jwtRefresh from "../jwtauth/jwtRefresh";
 import { db_error } from "../../app";
 import mongoose from "mongoose";
 import responseFunction from "../coms/apiResponse";
 import Comment from "../../models/post/comment";
 
 const router = Router();
-router.get ("/:target", async (req,res) => {//#CHECK DATABASE STATE AND WHETHER PROVIDED POST DATA IS VALID 
+router.get ("/:target", async (req,res) => {
```

Comment:
Codacy found an issue: [Async arrow function has a complexity of 7. Maximum allowed is 4.](https://app.codacy.com/gh/implude/local-community-backend/pullRequest?prid=6225533)